src/styles/DropdownMenu.module.css

```diff
@@ -8,9 +8,10 @@
 .dropdownContent {
   display: inline-block;
   background-color: var(--white);
-  position: absolute;
   top: 50px;
   box-shadow: 0 8px 16px 0 var(--black);
+  max-height: 250px;
+  overflow: auto;
 }
 
```

Comment:
where did you fix your footer to stick to the bottom of the page?

Reply:
I've set the Footer's properties "position: absolute" with "bottom: 0" and I added "position: relative" to the dropdown div. Before I made the dropdown content to display in a shorter list I tested the footer and it stuck to the bottom all the way down the cities list.

Reply:
I also added "position: relative", "min-height: 100vh" and "margin: 0" to the html body tag

Reply:
what do you need the position relative of the dropdown div for?

Reply:
Well, I had problems positioning the "Cities" button itself and the dropdown content, for example when my content appears the button shrunk to half its height, so I tried a lot of things and this is what I managed. Now, when I remove "position: relative" from dropdown div, the dropdown content doesn't appear on the screen when I click the button.

Reply:
okay